src/consensus/aft/raft.h

```diff
@@ -889,7 +889,7 @@ namespace aft
       }
       catch (const std::logic_error& err)
       {
-        LOG_FAIL_FMT(err.what());
+        LOG_FAIL_FMT("Error recv authenticated append entries: {}", err.what());
```

Comment:
These often end up saying "Error", which is redundant with "fail", and something that's essentially the name of the call. So I wonder if we shouldn't just use __PRETTY_FUNCTION__ instead and have a shortcut for that like LOG_FAIL_EXC(err).

Reply:
Good point. I'll add this!

Reply:
Maybe “Exception in {}: {}”, passing the function name and what().

Reply:
Done. I've also subsumed the existing `LOG_FAIL_FMT("Exception!"); LOG_DEBUG_FMT("Exception: {}", err.what());` pattern that we had into one macro.